app/run_localserver.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
```

Comment:
Let's remove `run_sample.sh` and keep only one example for Angular.

Reply:
Should we delete `app/client`?